src/components/search/SearchForm.vue

```diff
@@ -76,7 +82,14 @@
 
     components: {
       SearchQueryOptions,
-      FilterToggleButton
+      FilterToggleButton: () => import('./FilterToggleButton')
+    },
+
+    props: {
+      absoluteTopPositioned: {
+        type: Boolean,
+        default: false
```

Comment:
wouldn't it make more sense to have the 'default' be true for this?

Reply:
My idea was that the absolute positioned variant would definitely only be used once. If reused it would just be placed over the other. The relative positioned search bar could theoretically be reused.

Reply:
Righ, that's fine. I was thinking that the one in the top-nav would be used more frequently since it's on every page, therefore should be the default.